### PR TITLE
Cleanup README & detail the different install options

### PR DESCRIPTION
**Pull request by david-martin**
## What
Update the readme with more detailed info on the installation options

## Why
Make it easier to understand what's being done and what target makes sense to use

For anyone reviewing this, I'd recommend viewing the full fill in rendered markdown rather than the diff (https://github.com/RHCloudServices/kafka-monitoring-stuff/blob/4f5fc0bf429bb359bf78f6495f1b06ba1e0cef4d/README.md)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,67 +1,114 @@
...
+* *kafka-operator*: contains the Strimzi operator
+* *kafka-cluster*: contains the Kafka cluster
+* *managed-services-monitoring-global*: contains the global monitoring stack including Grafana, Thanos Receiver and Thanos Querier
+* *managed-services-monitoring-prometheus*: contains the on cluster Prometheus that scrapes Kafka metrics
+* *managed-services-monitoring-grafana*: contains the on cluster Grafana instance
+
+## Notes
+
+* The Grafana instances are protected by the OpenShift OAuth proxy. Sign in using an OpenShift account with permission to `get` `namespaces`.
+* To sign in to Grafana itself (once passed the proxy), use the credentials from the `grafana-admin-credentials` secret. This is only required if you want to modify dashboards (temporary as dashboards are persisted in GrafanaDashboard CRs & cannot be saved from the Grafana UI)
```
Comment (ppatierno) [inline README.md:114]: I would specify that the secret is in the `managed-services-monitoring-grafana` namespace, right?

**Review by ppatierno — APPROVED** [review]
LGTM. Just a couple of nits and a comment ...
Is it really necessary having `install` in all the targets? I guess that a `make strimzi/operator` or `make monitoring/cluster` makes anyway sense without the "install".

**Review by damienomurchu — APPROVED** [review]
Looks good, and agree with feedback from @ppatierno also

**Comment by david-martin** [comment]
> Is it really necessary having `install` in all the targets

Probably not, I can remove it.
I think it made sense when there were `uninstall` targets. Now we have `make clean`

File: README.md (hunk 1 above)
Comment (david-martin) [reply README.md:114]: Will do

File: README.md [hunk 2]
```diff
@@ -1,67 +1,114 @@
...
+* a kafka cluster
 
 ```sh
-$ make all
+make all
 ```
 
-__NOTE__: the grafana instances are protected via the OpenShift OAuth proxy
-__NOTE__: to gain admin access to grafana, login using the credentials from the `grafana-admin-credentials` secret
+### 2) Install *global* components *only*
```
[the remark this answers is not in the document]
Comment (david-martin) [reply README.md:36]: Lemme fix that up

**Comment by david-martin** [comment]
@damienomurchu @ppatierno 
I've pushed up some changes as per feedback.
Good to approve & merge?

File: install/Makefile [hunk 3]
```diff
@@ -54,33 +54,33 @@ uninstall/monitoring/cluster:
 	@oc delete namespace $(PER_CLUSTER_GRAFANA_NAMESPACE)
 	@echo "completed uninstallation of cluster monitoring stack"
 
-.PHONY: install/strimzi/operator
-install/strimzi/operator:
+.PHONY:strimzi/operator
+strimzi/operator:
```
Comment (ppatierno) [inline install/Makefile:58]: just realized that `install` would have sense because we also have `uninstall` for everything?

Comment (david-martin) [reply install/Makefile:58]: Let's keep it so 👍 
Undoing

Comment (david-martin) [reply install/Makefile]: Reverted

**Pull request merged**